confd/pkg/backends/calico/client.go

```diff
@@ -1349,6 +1350,15 @@ func (c *client) getNodeMeshPasswordKVPair(v3res *apiv3.BGPConfiguration, key in
 	}
 }
 
+func (c *client) getIgnoredInterfacesKVPair(v3res *apiv3.BGPConfiguration, key interface{}) {
+	ignoredInterfaces := getBGPConfigKey("ignored_interfaces", key)
+	if v3res != nil && v3res.Spec.IgnoredInterfaces != nil {
```

Comment:
nit: I don't expect this to happen, but I think we should also probably check if `v3rest.Spec` is `nil` too.